### PR TITLE
Enable Scala 2.13.0 in the linker and all the other artifacts.

### PR DESCRIPTION
**Pull request by sjrd**
This is the equivalent of #3702 for the `master` branch.

### CODE REVIEW

**Comment by sjrd** [comment]
Updated to match recent changes in #3702.

File: scala-test-suite/src/test/resources/2.13.0/BlacklistedTests.txt [hunk 1]
```diff
@@ -122,12 +172,20 @@ scala/util/matching/RegexTest.scala
 # Require strict-floats
 scala/math/BigDecimalTest.scala
 
-# Difference of getClass() on primitive values
-scala/collection/immutable/RangeTest.scala
-
-# Bugs
-scala/collection/convert/MapWrapperTest.scala
+# Fails for a BigDecimal range with augmented precision (might be an actual bug)
```
Comment (gzm0) [inline scala-test-suite/src/test/resources/2.13.0/BlacklistedTests.txt]: Same comment: Is this actually a bug?

Comment (sjrd) [reply scala-test-suite/src/test/resources/2.13.0/BlacklistedTests.txt]: Filed as #3706.

**Comment by sjrd** [comment]
Ping @gzm0. If possible I'd like to merge this before the open source spree of Scala Matsuri, happening on Thursday.

**Comment by sjrd** [comment]
Thanks :)

**Pull request merged**